webview/src/experiments/components/table/styles.module.scss

```diff
@@ -834,28 +834,6 @@ $badge-size: 0.85rem;
 
```

Comment:
The table bottom border is showing on the table:

<img width="614" alt="image" src="https://user-images.githubusercontent.com/43496356/234041267-0d55bc6d-eb51-4482-9f65-ef7e9265791f.png">

Though in my opinion, this removed border is minor and we can just keep the border on all rows if it's hard to implement. If we decide this, we'll need to remove the corresponding CSS though.